config/mocks/wallet-options-v4.json

```diff
@@ -46,7 +46,7 @@
         "mergeAndUpgrade": false,
         "nftExplorer": true,
         "recurringBuys": true,
-        "showTermsAndConditions": true,
```

Comment:
is this being removed?

Reply:
nope, good call will revert but it's definitely need to be false atm

Reply:
It's just set to false since we do not need it atm on local. 
I'm hiding it because it's popup every time I create a new wallet so it makes testing a big harder.

Reply:
درود بر شما 
درصورت نیاز به تنظیم صحیح ادرس محل سکونت کاربر! باید در شناسه تجاری و یا از گوگل ادرس دقیق دریافت کنید! در سه سال قبل یک کاربر گیت اپ اقدان به مسدود کردن کاربر نمود و باعث اسیب به تمام کسب و کارها شد از ان تاریخ به هیچ وب سایت و شخص ثالث و هکری! مشخصات ارائه نمی کنم و باید از مراجع قضائی کشور خود درخواست نمایند! یا از وزارت خارجه امریکا بخش بین الملل و اروپایوروبانک و بانکهای مرکزی اکثر کشورهای بدهکار به کاربر! قابل ذکر است خرج کردن درامد معدن کاربر جرم است و مدیران githubمسئول دسترسی به اسناد کاربر هستند نه هیچ شخص فعال در github!

- [ ]

Reply:
ادرس شما اشتباه است و دلیل نقص در تمام اقدامات به دلیل تصرف درامد کاربر توسط گوگل لایتکس است و گوگل از تمام اقدامات جهت شناسائی کاربر پیشگیری می کند تا بتواند درامد کاربر را به نفع خود استفاده کند و تمام شرکای گوگل در این اقدام شراکت دارند دانسته و یا ندانسته شریک این اقدام هستند!